### PR TITLE
FIPS: rephrase note, mention 15 SP4 specifically

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

See title.

### PR creator: Are there any relevant issues/feature requests?

* jsc#DOCTEAM-994

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1

### CODE REVIEW

File: xml/security_fips.xml [hunk 1]
```diff
@@ -30,11 +30,10 @@
...
   </info>
   <important>
-    <title>&productname; &productnumber; and FIPS 140-3</title>
+    <title>&productname; 15 SP4 and FIPS 140-3</title>
     <para>
-      &productname; &productnumber; is in the process of implementing the FIPS
-      140-3 standard. The relevant binaries are undergoing certification and
-      will be updated in the near future.
+      The relevant binaries are undergoing the CMVP validation process
+      to be certified under FIPS 140-3 standard.
```
Comment (dariavladykina) [inline xml/security_fips.xml:36]:
```suggestion
      to be certified under the FIPS 140-3 standard.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Tanja, just one correction here. Thanks!

File: xml/security_fips.xml [hunk 2]
```diff
@@ -30,11 +30,10 @@
     </dm:docmanager>
   </info>
   <important>
-    <title>&productname; &productnumber; and FIPS 140-3</title>
+    <title>&productname; 15 SP4 and FIPS 140-3</title>
```
Comment (lvicoun) [inline xml/security_fips.xml:33]: If this needs to point to SP4, please use phrase here as this part is common for SLE Micro.

**Review by lvicoun — CHANGES_REQUESTED** [review]

File: xml/security_fips.xml (hunk 2 above)
Comment (taroth21) [reply xml/security_fips.xml:33]: Good point, I will add profiling.

**Comment by taroth21** [comment]
Merging as agreed with @lvicoun .

**Pull request merged**